internal/infrastructure/db/pg/migrations/20220223123306_market.up.sql

```diff
@@ -1,10 +1,8 @@
 CREATE TABLE market (
     market_id SERIAL,
-    account_index int NOT NULL,
     provider_name varchar(264) NOT NULL,
     url          varchar(264) NOT NULL,
-    credentials  varchar(264) NOT NULL,
     base_asset    varchar(264) NOT NULL,
     quote_asset   varchar(264) NOT NULL,
-    PRIMARY KEY(account_index, provider_name)
+    PRIMARY KEY(provider_name, base_asset, quote_asset)
```

Comment:
not sure provider name is safe as unique ID, I guess URL is more unique

Reply:
Done